clowdapp.yaml

```diff
@@ -114,8 +116,9 @@ objects:
           value: "${SETTINGS__FORCE_IMPORT_SSGS}"
         - name: SETTINGS__REPORT_DOWNLOAD_SSL_ONLY
           value: ${SETTINGS__REPORT_DOWNLOAD_SSL_ONLY}
+        - name: SETTINGS__KAFKA__SECURITY_PROTOCOL
+          value: ${SETTINGS__KAFKA__SECURITY_PROTOCOL}
     - name: compliance-backend
-      # TODO: Check how to auto-scale with Clowder
```

Comment:
Unrelated line is removed. This TODO is still valid.

Reply:
you really cannot autoscale with Clowder, we saw that, there's no way to do it as of today, so we hardcoded the number of replicas, that's why I removed it. 

If you still think it's valid I can put it back.

Reply:
I'd prefer to keep it so we remember to integrate with the feature once Clowder implements it, but I don't feel strongly.